source/ranges.tex

```diff
@@ -3202,6 +3210,430 @@
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{to-unsigned-like}(\exposid{bound_});}
+\end{itemdescr}
+
+\rSec3[range.repeat.iterator]{Class template \tcode{repeat_view::\exposid{iterator}}}
+
+\begin{codeblock}
+namespace std::ranges {
+  template<@\libconcept{move_constructible}@ W, @\libconcept{semiregular}@ Bound = unreachable_sentinel_t>
+    requires (is_object_v<W> && @\libconcept{same_as}@<W, remove_cv_t<W>> &&
+              (@\exposid{is-integer-like}@<Bound> || @\libconcept{same_as}@<Bound, unreachable_sentinel_t>))
+  class repeat_view<W, Bound>::@\exposid{iterator}@ {
+  private:
+    using @\exposid{index-type}@ =                  // \expos
+      conditional_t<@\libconcept{same_as}@<Bound, unreachable_sentinel_t>, ptrdiff_t, Bound>;
+    const W* @\exposid{value_}@ = nullptr;          // \expos
+    @\exposid{index-type}@ @\exposid{current_}@ = @\exposid{index-type}@(); // \expos
+
+    constexpr explicit @\exposid{iterator}@(const W* value, @\exposid{index-type}@ b = @\exposid{index-type}@());   // \expos
+
+  public:
+    using iterator_concept = random_access_iterator_tag;
+    using iterator_category = random_access_iterator_tag;
+    using value_type = W;
+    using difference_type = conditional_t<@\exposid{is-signed-like}@<@\exposid{index-type}@>,
+        @\exposid{index-type}@,
+        @\placeholdernc{IOTA-DIFF-T}@(@\exposid{index-type}@)>;
+
+    @\exposid{iterator}@() = default;
+
+    constexpr const W& operator*() const noexcept;
+
+    constexpr @\exposid{iterator}@& operator++();
+    constexpr @\exposid{iterator}@ operator++(int);
+
+    constexpr @\exposid{iterator}@& operator--();
+    constexpr @\exposid{iterator}@ operator--(int);
+
+    constexpr @\exposid{iterator}@& operator+=(difference_type n);
+    constexpr @\exposid{iterator}@& operator-=(difference_type n);
+    constexpr const W& operator[](difference_type n) const noexcept;
+
+    friend constexpr bool operator==(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y);
+    friend constexpr auto operator<=>(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y);
+
+    friend constexpr @\exposid{iterator}@ operator+(@\exposid{iterator}@ i, difference_type n);
+    friend constexpr @\exposid{iterator}@ operator+(difference_type n, @\exposid{iterator}@ i);
+
+    friend constexpr @\exposid{iterator}@ operator-(@\exposid{iterator}@ i, difference_type n);
+    friend constexpr difference_type operator-(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y);
+  };
+}
+\end{codeblock}
+
+\indexlibraryctor{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr explicit @\exposid{iterator}@(const W* value, @\exposid{index-type}@ b = @\exposid{index-type}@());
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+If \tcode{Bound} is not \tcode{unreachable_sentinel_t}, $\tcode{b} \ge 0$.
+
+\pnum
+\effects
+Initializes \exposid{value_} with \tcode{value} and
+\exposid{bound_} with \tcode{b}.
```

Comment:
_`iterator`_ does not have a member _`bound_`_, should this be _`current_`_? I suspect this is a typo in the paper.

Reply:
@jwakely, @CaseyCarter: or would you like to fix this immediately?

Reply:
Actually, since this seems like a simple error, let's just fix it right now.